src/App.js

```diff
@@ -13,10 +15,125 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Goods 1</h1>
-  </div>
-);
+class App extends React.Component {
+  state = {
+    listOfGoods: [...goodsFromServer],
```

Comment:
название похоже больше на компонент
```suggestion
    goods: [...goodsFromServer],
```